src/api/client/__tests__/Settlements.ts

```diff
@@ -1,12 +1,13 @@
 import BigNumber from 'bignumber.js';
 
 import { addInstructionTransformer, Context, TransactionQueue, Venue } from '~/internal';
-import { Settlements } from '~/Settlements';
 import { dsMockUtils, entityMockUtils, procedureMockUtils } from '~/testUtils/mocks';
 import { Mocked } from '~/testUtils/types';
 import { Instruction, VenueType } from '~/types';
 import { InstructionAffirmationOperation } from '~/types/internal';
 
+import { Settlements } from '../Settlements';
```

Comment:
(nit) I thought we mostly used `~` as convention.

I am not against relative paths, but I like the consistency of `~` everywhere.

Reply:
Fair enough, I'll fix it